.travis.yml

```diff
@@ -17,9 +17,9 @@ before_script:
 script:
  - composer test-ci
  - vendor/bin/phpcs --config-set ignore_warnings_on_exit 1
- - vendor/bin/phpcs --standard=psr2 src
- - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then vendor/bin/phpcs --standard=phpcs.xml src --ignore=tests/Sniffs; fi
- - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then vendor/bin/phpcs --standard=codor.xml src -spn; fi
```

Comment:
None of these condition will evaluate to `true`, but `phpcs` is run using these rules in the `test` script in [`composer.json`](https://github.com/bmitch/churn-php/blob/master/composer.json#L58-L61).